src/components/routerLink.browser.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { mount } from '@vue/test-utils'
+import { expect, test, vi } from 'vitest'
+import { h } from 'vue'
+import routerLink from '@/components/routerLink'
+import { Route } from '@/types'
+import { component, createRouter } from '@/utilities'
```

Comment:
no need to do anything now, just thinking. I don't think testHelpers was supposed to be included in barrel, I likely added it with my barrel vscode plugin. Might have to update this import if/when it gets removed again

Reply:
We should really move test stuff into its own folder